### ISSUE TITLE
textri glitch. Textured triangle disappears.

### ISSUE THREAD

**Issue by Tinchetsu** [nesbox/TIC-80#458]
This happens in 0.50.1.
I'm trying to make some basic 3D games, rendering lowpoly models with textri, and I found this glitch.
Textured triangles created with textri, sometimes dissapears if some vertices goes offscreen.

Cart used for test: https://www.dropbox.com/s/9946yswemx4cumi/triglitch.tic?dl=0
(Move vertex with arrows, change selected vertex with z)

Attached gif with the issue.
![glitch](https://user-images.githubusercontent.com/10901731/33810071-328af81c-dddf-11e7-86aa-706a03800be1.gif)

**Comment by Tinchetsu** [comment]
After some more test, I found that is not about being offscreen. A textri with a height bigger than TIC80 screen height, is rejected.

After checking the source a bit, I found this line in tic.c:
https://github.com/nesbox/TIC-80/blob/master/src/tic.c#L915

In the method ticTexLine, "texlines" are rejected if the textri height is bigger than TIC80_HEIGHT. Not sure why is that, maybe to fix some crash?

A better check would be: to check if the line that is going to be rendered is in screen.
